src/dc_receive_imf.rs

```diff
@@ -1441,7 +1441,9 @@ async fn create_or_lookup_group(
             return Ok(None);
         }
 
-        let grpname = mime_parser.get_header(HeaderDef::ChatGroupName).unwrap();
+        let grpname = mime_parser
+            .get_header(HeaderDef::ChatGroupName)
+            .context("we checked that Chat-Group-Name is there, but it vanished")?;
```

Comment:
sounds quite long would rather go for: 
```suggestion
            .context("Chat-Group-Name vanished?!")?;
```

Reply:
Changed it to "Chat-Group-Name vanished". Anyhow adds ":" between contexts, so "?!" does not fit here.